packages/css/src/details.css

```diff
@@ -110,10 +105,34 @@
     interpolate-size: allow-keywords;
   }
 
+  &::details-content {
```

Comment:
can do:
```suggestion
  &::details-content,
  &::part(details-content) {
    ...
  }
```
instead of repeating same twice? ☺️

Reply:
Yeah i couldn't get that to work on Firefox.

Reply:
hm, if it does not work, it might be because firefox does not understand one of the selectors. We can then use `&:is(::details-content, ::part(details-content))` instead, as `:is()` is forgiving if one of the selectors is unknown ☺️

Reply:
Cool I’ll try that.

Reply:
:is does not select pseudo elements 😞

Reply:
Waaat. Æsj. Learning something new everyday 🤷‍♂️